src/django/api/models.py

```diff
@@ -93,24 +93,26 @@ def create_superuser(self, email, password, **extra_fields):
         return self._create_user(email, password, **extra_fields)
 
 
-class ContributorManager(models.Manager):
+class ContributorManager(models.QuerySet):
```

Comment:
No, it works fine the way we had it before, but implementing it as a `QuerySet` makes it more flexible / less brittle, since you no longer have to start your ORM chain with the custom manager method: https://docs.djangoproject.com/en/4.0/topics/db/managers/#creating-a-manager-with-queryset-methods